kolibri/plugins/coach/assets/src/app.js

```diff
@@ -68,12 +68,26 @@ class CoachToolsModule extends KolibriModule {
           },
         },
         {
+          name: PageNames.COACH_EXERCISE_RENDER_PAGE,
+          path: '/:user_id/:content_id/exercise-render',
+          handler: (toRoute, fromRoute) => {
+            actions.showCoachExerciseRenderPage(store, toRoute.params);
```

Comment:
Pass in `user_id` and `content_id` params separate instead of just an object to make it more explicit.